include/sys/types.h

```diff
@@ -2,9 +2,9 @@
 #define _SYS_TYPES_H_
 
 #include <sys/cdefs.h>
-#include <endian.h>
 #include <stdint.h> /* uint*_t, int*_t */
 #include <stddef.h> /* offsetof, NULL, ptrdiff_t, size_t, etc. */
+#include <endian.h>
```

Comment:
Moved by mistake?

Reply:
Not really, it was because of a cyclic dependency: `sys/types.h` -> `endian.h` -> `sys/endian.h` -> `sys/types.h`. Now I've fixed it by not including `sys/endian.h` in `sys/endian.h`.